libs/sdk-backend-bear/src/backend/index.ts

```diff
@@ -270,6 +272,10 @@ export class BearBackend implements IAnalyticalBackend {
         return this.authProvider.deauthenticate(this.getAuthenticationContext());
     }
 
+    public organization(organizationId: string): IOrganization {
+        return new BearOrganization(this.sdk, organizationId);
```

Comment:
I think you need to pass the `this.authApiCall` guard. the guard ensures proper integration with authentication providers. the new service should be using the guard (see other services) and make the sdk call through it. the guard ensures that if API call ends with 401 (no TT and no SST) then the auth provider flow starts.

Reply:
It is not necessary as the the API call does not require authentication but I added it to better compatibility with future extensions of the service as per Slack discussion.